### PR TITLE
Update condition to display new label for content imports

### LINKED ISSUE #9415: Confusing UI when importing content over a network
## Observed behavior
When importing from someone over the network, if they have a lower version of the same channel, some items may show up with a “New” flag on resources that are actually old. This occurred an QA channel versions 95 → 90.

## Errors and logs
None

## Expected behavior
Older versions of the channel should not be flagged as "new"

## User-facing consequences
Confusion!

## Steps to reproduce
Import from someone over the network, when the person you are importing from has an older version of the channel than you currently have on your device

## Context
Single-user-sync context 
Jamie	Ubuntu 20.04	Chrome
Jacob	Fedora 34	Chrome

### PR DESCRIPTION
**Pull request by LianaHarris360**
## Summary

This pull request includes a modification where the string "Resources recently updated" is used instead of the label "New" when importing channel resources on the device to resolve any confusion surrounding to recently imported resources that are considered "new" on the device, but could be imported from older versioned channels.

Before:
labelbefore

After:

afterlabel

## References

Fixes #9415 

## Reviewer guidance

Import from someone over the network, making sure that the person you are importing from has an older version of the channel compared to the one currently installed on your device. The label 'Resources recently updated' should be displayed.

----

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Review by rtibbles — COMMENTED** [review]
Code changes makes perfect sense, needs a manual verification, and also holding off approval as this is targeted for the patch.

**Comment by rtibbles** [comment]
After consultation with @marcellamaki I am retargeting this to 0.16.0, so this can be merged once manual QA is complete.

**Comment by marcellamaki** [comment]
@radinamatic @pcenov this is ready for manual QA review. Thank you!

**Comment by pcenov** [comment]
Hi @LianaHarris360 for some reason following the steps from the Reviewer guidance section I am actually seeing the "New" label when importing from an older version of the channel. See the following video:

https://github.com/learningequality/kolibri/assets/79847249/68f4c5a6-78d2-4882-8b3b-183e92d10cae

**Comment by LianaHarris360** [comment]
Hi @pcenov, I noticed that when choosing the option to import from another server, the channel version it is selecting from has the same version as the channel on the device ("Version 19"), although the second server on the right side of the screen lists the channel as "Version 8". Since the version is the same when selecting resources, the `New` label is displayed to indicate that resources were added or updated until the task is cleared, which seems to be the expected behavior. Perhaps in the future there should be a distinction between a newly added channel/version update and newly added resources for more clarity.

**Comment by pcenov** [comment]
Hi @LianaHarris360 but that just makes the problem bigger than I initially thought. I've just double checked this with a newly created channel the first version of which I imported to the server and the second version of which I imported to the other device. 
When I attempt to import something from the server to the device it incorrectly states the version of the channel to be the same as the one on the device. This prevents me from testing the scenario outlined in the 'Reviewer guidance' as it always treats the version that's on the server as being the same as the one on the device... :)

File: kolibri/plugins/device/assets/src/views/ManageContentPage/index.vue
Comment (rtibbles) [inline kolibri/plugins/device/assets/src/views/ManageContentPage/index.vue]: Oh wait - is this logic the correct way around? `channelVersion` is the version of the channel on the remote, right? So we'd want to flag it in the case that `match.version < channelVersion`?

**Comment by LianaHarris360** [comment]
@pcenov It seems that the local channel version is supposed to be displayed, unless it is not available, because the version of the channel on the local device is being used to display the channel contents locally, not the remote version. From my understanding, the `New` label should be displayed if the remote channel version is newer or if there are newly added resources, even if those resources come from an older version channel. Also, there was some logic that was backwards that has now been updated, the code should work as intended now.

**Comment by pcenov** [comment]
Thanks @LianaHarris360 I can only say that the label "New" is being always displayed when importing content either from Studio or from an older version of the channel from a local device. When importing from an older channel version on a local device the version of the channel is shown as being the same as the one on my device (which is confusing and incorrect). However if I am directly importing the channel for the first time from the local device then it shows the correct version. Here's a video illustrating what I am doing and seeing:

https://github.com/learningequality/kolibri/assets/79847249/322b57d9-911a-41c3-a623-14dac72a0e72

Based on that I still can't understand how to test the issue described in https://github.com/learningequality/kolibri/issues/9415 so some guidance on that will be much appreciated.

**Comment by LianaHarris360** [comment]
Ah, yes, I understand your point. When selecting resources to import, if the displayed channel version always matches the one on the device, then the label "New" will always be displayed after importing resources. I'll have to defer to Richard to provide a more thorough explanation as to why the device's channel version is shown instead of the remote version. 

Since the device's channel version being displayed is intentional, perhaps the primary issue is not with the code logic, but rather with the labeling of "New", which may cause confusion among users when added resources are not necessarily more current. cc @rtibbles @marcellamaki @radinamatic

**Review by rtibbles — APPROVED** [review]

**Comment by rtibbles** [comment]
Merging - any further iteration on the strings can happen in string review process for 0.17.

**Pull request merged**
